src/components/MenuBar/FlipCameraButton/FlipCameraButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import FlipCameraIosIcon from '@material-ui/icons/FlipCameraIos';
+import { IconButton } from '@material-ui/core';
+import useVideoContext from '../../../hooks/useVideoContext/useVideoContext';
+
+export default function FlipCameraButton() {
+  const {
+    room: { localParticipant },
+    localTracks,
+    getLocalVideoTrack,
+  } = useVideoContext();
+  const [supportsFacingMode, setSupportsFacingMode] = useState<Boolean | null>(null);
+  const videoTrack = localTracks.find(track => track.name === 'camera');
+  const facingMode = videoTrack?.mediaStreamTrack.getSettings().facingMode;
+
+  useEffect(() => {
+    // The 'supportsFacingMode' variable determines if this component is rendered
+    // If facingMode is true, we will set supportsFacingMode to true.
```

Comment:
Nit: To be more accurate, should this say "If facingMode is available (or something similar),"? Because facingMode is not boolean right?

Reply:
Good call - updated :+1: